### ISSUE TITLE
React build failed with typescript-eslint version 5.33.0

### ISSUE THREAD

**Issue by patcrama** [typescript-eslint/typescript-eslint#5447]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.
- [X] I have [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Abug+label%3A%22package%3A+eslint-plugin%22) and found none that matched my issue.
- [X] I have [read the FAQ](https://typescript-eslint.io/docs/linting/troubleshooting) and my problem is not listed.

### Playground Link

no link

### Repro Code

```typescript
All of sudden today I started to have build error with following messages, please help me to fix this

[INFO] error Couldn't find any versions for "@typescript-eslint/scope-manager" that matches "5.33.0"
[INFO] error Couldn't find any versions for "@typescript-eslint/scope-manager" that matches "5.33.0"info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
[INFO] Error: Couldn't find any versions for "@typescript-eslint/types" that matches "5.33.0"
```

### ESLint Config

```javascript
module.exports = {
  parser: "@typescript-eslint/parser",
  rules: {
    "@typescript-eslint/<rule-name>": ["error", ...<options>],
  },
};
```

### tsconfig

```jsonc
{
  "compilerOptions": {
    // ...
  }
}
```

### Expected Result

build should run fine with Eslint but it's giving error

### Actual Result

build failed with 5.33.0

### Versions

 "devDependencies": {
    "@types/react-router-dom": "^5.3.2",
    "@types/react-router-redux": "^5.0.21",
    "@types/react-table": "^7.7.9",
    "@types/styled-components": "^5.1.25",
    "@typescript-eslint/scope-manager": "5.32.0",
    "@typescript-eslint/typescript-estree": "5.32.0",
    "@typescript-eslint/types":"5.32.0",
    "eslint-plugin-react-hooks": "^4.6.0",
    "sass": "^1.42.1",
    "typescript": "^4.7.4"
  }

**Comment by patcrama** [comment]
Can some body please help me as this is blocking to build my react project

**Comment by bradzacher** [comment]
It's all published on NPM just fine.
I can see the versions listed just fine on NPM.

Sometimes npm can lag behind a little after a new release. Just give it some time and try again.
